scripts/volmount/crypt

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+op=$1
+disk_path=$2
+volume=$3
+loopdev=$4
+
+do_mount_disk()
+{
+	mkdir -p /data/caam
+	/lib/pv/volmount/caam-keygen import /data/caam/key importKey
+	cat /data/caam/importKey | keyctl padd logon logkey2: @u
+
+	dmsetup -v create encrypted --table "0 $(blockdev --getsz $3) crypt capi:tk(cbc(aes))-plain :36:logon:logkey2: 0 $3 0 1 sector_size:512"
```

Comment:
when and where is that disk formatted/created first time it is encountered?

Reply:
Device file creation is handled in the script for 1st boot now.